inference-engine/src/offline_transformations/include/mask_attribute.hpp

```diff
@@ -0,0 +1,139 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief Defines primitives priority attribute
+ * @file primitives_priority_attribute.hpp
+ */
+
+#pragma once
+
+#include <assert.h>
+#include <functional>
+#include <memory>
+#include <string>
+#include <set>
+
+#include <ngraph/node.hpp>
+#include <ngraph/variant.hpp>
+#include <transformations_visibility.hpp>
+
+namespace ngraph {
+
+/**
+ * @ingroup ie_runtime_attr_api
+ * @brief each element in vector represents dimension and each element
+ * in set is an id of dimensions which contains zeros.
+ */
+class TRANSFORMATIONS_API Mask : public std::vector<std::set<uint64_t>>,
```

Comment:
export macro is not needed for static libraries.

Reply:
Done.